docs/voiceover-guidelines.md

```diff
@@ -1,61 +1,179 @@
+<img src="images/voiceover-guidelines/textfield-filled-with-label.png" width="400">
 
-![](images/voiceover-guidelines.png)
+With this, VoiceOver will read the label (e.g. `"First name"`) and the value (e.g. `"John"`).
 
-The reading order is expected to be spoken as, "Subtotal: nine hundred ninety-nine dollars and ninety nine cents". In this case, VoiceOver defaults to reading the first item in each horizontal stack view, followed by the second. The default order speaks a list of titles: "Subtotal - Discount - Shipping - Taxes" and so on, followed by the values: "$999.99, -$601.00, $0.01, $333.33", making the information difficult to comprehend.
+### <a name="grouping-elements"></a>Grouping Elements
 
-Use `.accessibilityElements` on the parent view to list the desired reading order of the subviews.
+If a group of elements represent a single unit of information, consider grouping them into one accessibility element. This helps reduce clutter and makes your app easier to understand and navigate. 
+
+Take the following custom `UITableViewCell` as an example. It has at least 5 accessible elements. 
+
+<img src="images/voiceover-guidelines/group-elements-before.png" width="320">
+
+Since there are potentially more cells like this in the table, it would be very easy for a VoiceOver user to lose context. To improve this, you can:
+
+- Group the elements together by concatenating the information in the `UITableViewCell`'s `accessibilityLabel`.
+- And make the child elements inaccessible by setting their `isAccessibilityElement` to `false`.
+
+```swift
+class CustomCell: UITableViewCell {
+    override var accessibilityLabel: String? {
+        get {
+            let format = "Post by %@, from %@. %@. %@. %@. Excerpt. %@."
+            return String(format: format,
+                          author,
+                          blogName,
+                          datePublished,
+                          isFollowing ? "Following" : "",
+                          title,
+                          excerpt)
+        }
+        set { }
+    }
+}
 ```
+
+When the cell is focused, VoiceOver would then speak something like this:
+
+```
+Post by Carolyn Wells, from Discover. 6 days ago. Following. 
+Around the World with WordPress: Jamaica. 
+Excerpt. Today, we’re highlighting five sites from the island paradise of Jamaica.
+```
+
+- Prefer to place the most important elements first. The VoiceOver user can prefer to skip if they've already listened to what they need. This is why we placed the `excerpt` last in the example.
+- Don't forget the periods when concatenating. They make VoiceOver pause which helps in comprehension.
```

Comment:
```suggestion
- Don't forget the periods when concatenating. They make VoiceOver pause which helps to prevent incomprehension.
```
Since incomprehension is a word, "in comprehension" might be misrepresented.